src/components/books.jsx

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable max-len */
```

Comment:
- [ ] `required`
- [ ] please enable the linters because it defeats the purpose of installing linters